display.launch が動かない

次のコマンドでdisplay.launchを実行しましたが、
エラーが表示され、モデルが正常に描画されませんでした。

```sh
$ roslaunch crane_x7_description display.launch
```
実行環境はUbuntu18.04、ROS Melodicです。

エラーログ
```sh
roslaunch crane_x7_description display.launch 
... logging to /home/shotaak/.ros/log/b8a8d610-4e0a-11ea-a521-b88584a3c7d9/roslaunch-shotaak-Alienware-Aurora-R7-12907.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

xacro: in-order processing became default in ROS Melodic. You can drop the option.
Deprecated: xacro tag 'crane_x7' w/o 'xacro:' xml namespace prefix (will be forbidden in Noetic)
when processing file: /home/shotaak/catkin_ws/src/crane_x7_ros/crane_x7_description/urdf/crane_x7.urdf.xacro
Use the following command to fix incorrect tag usage:
find . -iname "*.xacro" | xargs sed -i 's#<\([/]\?\)\(if\|unless\|include\|arg\|property\|macro\|insert_block\)#<\1xacro:\2#g'

started roslaunch server http://shotaak-Alienware-Aurora-R7:44771/

SUMMARY
========

PARAMETERS
 * /rate: 10
 * /robot_description: <?xml version="1....
 * /rosdistro: melodic
 * /rosversion: 1.14.3
 * /source_list: ['joint_states_so...
 * /use_gui: True

NODES
  /
    joint_state_publisher (joint_state_publisher/joint_state_publisher)
    robot_state_publisher (robot_state_publisher/robot_state_publisher)
    rviz (rviz/rviz)

auto-starting new master
process[master]: started with pid [12930]
ROS_MASTER_URI=http://localhost:11311

setting /run_id to b8a8d610-4e0a-11ea-a521-b88584a3c7d9
process[rosout-1]: started with pid [12941]
started core service [/rosout]
process[joint_state_publisher-2]: started with pid [12944]
process[robot_state_publisher-3]: started with pid [12945]
process[rviz-4]: started with pid [12946]
[WARN] [1581561855.692786]: The 'use_gui' parameter was specified, which is deprecated.  We'll attempt to find and run the GUI, but if this fails you should install the 'joint_state_publisher_gui' package instead and run that.  This backwards compatibility option will be removed in Noetic.
[ERROR] [1581561855.693473]: Could not find the GUI, install the 'joint_state_publisher_gui' package
[joint_state_publisher-2] process has died [pid 12944, exit code 1, cmd /opt/ros/melodic/lib/joint_state_publisher/joint_state_publisher __name:=joint_state_publisher __log:=/home/shotaak/.ros/log/b8a8d610-4e0a-11ea-a521-b88584a3c7d9/joint_state_publisher-2.log].
log file: /home/shotaak/.ros/log/b8a8d610-4e0a-11ea-a521-b88584a3c7d9/joint_state_publisher-2*.log
```
![スクリーンショット_2020-02-13_11-48-01](https://user-images.githubusercontent.com/18494952/74396901-b63fa600-4e56-11ea-9980-04f625df2324.png)

--- a/crane_x7_description/package.xml
+++ b/crane_x7_description/package.xml
@@ -17,4 +17,5 @@
 
   <run_depend>rospy</run_depend>
   <run_depend>std_msgs</run_depend>
+  <run_depend>joint_state_publisher_gui</run_depend>
 </package>

